spec/api-app-spec.js

```diff
@@ -425,6 +425,52 @@ describe('app module', () => {
       w.loadURL('about:blank')
       w.webContents.executeJavaScript(`require('electron').remote.getGlobal('test')`)
     })
+
+    it('should emit remote-get-builtin event when remote.getBuiltin() is invoked', (done) => {
+      app.once('remote-get-builtin', (event, webContents, moduleName) => {
+        expect(webContents).to.equal(w.webContents)
+        expect(moduleName).to.equal('app')
```

Comment:
`expect()` in a callback will cause the test to hang and timeout if it fails, i think? either a) refactor this test to use `async` and `emittedOnce` and so on, or b) wrap this in `try { ...; done() } catch (e) { done(e) }`

Reply:
made async